src/main/java/org/gitlab4j/api/ProtectedBranchesApi.java

```diff
@@ -228,6 +230,7 @@ public ProtectedBranch protectBranch(Object projectIdOrPath, String branchName,
             allowedToMerge.getForm(formData, "allowed_to_merge");
         if (allowedToUnprotect != null)
             allowedToUnprotect.getForm(formData, "allowed_to_unprotect");
+        //FIXME: cannot test, having CE only to test
```

Comment:
I don't think we need this FIXME line in the code.

```suggestion
```

If you know how we could have access to a Testing-Ultimate version, that would be great. I have this on my list, but I would need to ask somebody at GitLab.